_docs/en/03-add-action-for-mail-compose.md

```diff
@@ -100,4 +98,4 @@ Congratulations! You successfully added a button to the Mail Compose view, and r
 - This action uses the `path` property to specify were it is displayed in the Verse UI
 - Using `mail.compose` as the path will cause the action to be displayed in the toolbar in the mail compose window
 - The action will also be available when replying to or forwarding a mail
-- The action is always displayed in the overflow menu of the toolbar
\ No newline at end of file
+- The action is always displayed in the overflow menu of the toolbar
```

Comment:
To avoid confusion, better change _overflow_ to _"More actions" dropdown_, to be consistent with Verse UI